src/main/groovy/org/edx/jenkins/dsl/JenkinsPublicConstants.groovy

```diff
@@ -15,6 +15,8 @@ class JenkinsPublicConstants {
     }
   }
 
+  public static final String JENKINS_PUBLIC_GITHUB_BASEURL = "http://github.com/"
```

Comment:
Should we be using http or https? What are the advantages/disadvantages to either of those choices?